python/Pipfile

```diff
@@ -20,6 +20,7 @@ pytest-pudb = "*"
 pytest-mock = "*"
 twine = "*"
 pillow = "*"
+v = {editable = true,version = "*"}
```

Comment:
was this intentional?

Reply:
Something stuck it in so I thought it should belong. I can remove it

Reply:
`pipenv uninstall v --dev`